From 535c7e38d7cd5ccdb418f35b42e11c3ea41a6cce Mon Sep 17 00:00:00 2001
Message-ID: <535c7e38d7cd5ccdb418f35b42e11c3ea41a6cce.1709812934.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1709812934.git.sharlatanus@gmail.com>
References: <cover.1709812934.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 29 Feb 2024 08:18:09 +0000
Subject: [PATCH 6/7] gnu: go-github-com-beorn7-perks-quantile: Fix
 indentation.

* gnu/packages/syncthing.scm (go-github-com-beorn7-perks-quantile): Fix
indentation.

Change-Id: Id159b4314fc25ecebc0737221995823e28045626
---
 gnu/packages/syncthing.scm | 41 +++++++++++++++++++-------------------
 1 file changed, 21 insertions(+), 20 deletions(-)

diff --git a/gnu/packages/syncthing.scm b/gnu/packages/syncthing.scm
index 4d1e1c9ada..cf417fbe91 100644
--- a/gnu/packages/syncthing.scm
+++ b/gnu/packages/syncthing.scm
@@ -707,28 +707,29 @@ (define-public go-github-com-syncthing-notify
       (license expat))))
 
 (define-public go-github-com-beorn7-perks-quantile
-    (package
-      (name "go-github-com-beorn7-perks-quantile")
-      (version "1.0.1")
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                       (url "https://github.com/beorn7/perks")
-                       (commit (string-append "v" version))))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "17n4yygjxa6p499dj3yaqzfww2g7528165cl13haj97hlx94dgl7"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/beorn7/perks/quantile"
-         #:unpack-path "github.com/beorn7/perks"))
-      (synopsis "Compute approximate quantiles over an unbounded data stream")
-      (description "Perks contains the Go package @code{quantile} that computes
+  (package
+    (name "go-github-com-beorn7-perks-quantile")
+    (version "1.0.1")
+    (source (origin
+              (method git-fetch)
+              (uri (git-reference
+                    (url "https://github.com/beorn7/perks")
+                    (commit (string-append "v" version))))
+              (file-name (git-file-name name version))
+              (sha256
+               (base32
+                "17n4yygjxa6p499dj3yaqzfww2g7528165cl13haj97hlx94dgl7"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/beorn7/perks/quantile"
+       #:unpack-path "github.com/beorn7/perks"))
+    (synopsis "Compute approximate quantiles over an unbounded data stream")
+    (description
+     "Perks contains the Go package @code{quantile} that computes
 approximate quantiles over an unbounded data stream within low memory and CPU
 bounds.")
-      (home-page "https://github.com/beorn7/perks")
-      (license expat)))
+    (home-page "https://github.com/beorn7/perks")
+    (license expat)))
 
 (define-public go-github-com-matttproud-golang-protobuf-extensions-pbutil
   (let ((commit "c12348ce28de40eed0136aa2b644d0ee0650e56c")
-- 
2.41.0

